monitor-setup/checks.json

```diff
@@ -1,643 +1,656 @@
+      "msp_encryption": "SSL_TLS",
+      "msp_threshold": 40,
+      "msp_headers": "",
+      "msp_script": "",
+      "msp_version": 2,
+      "msp_sensitivity": 2,
+      "msp_num_retries": 2,
+      "msp_use_ip_version": "",
+      "msp_uptime_sla": "0.9900",
+      "msp_response_time_sla": "2.200",
+      "msp_notes": "",
+      "msp_include_in_global_metrics": true,
+      "stats_url": "https://uptime.com/api/v1/checks/1212958/stats/",
+      "analysis_url": "https://uptime.com/api/v1/checks/1212958/analysis/",
+      "alerts_url": "https://uptime.com/api/v1/alerts/?check_pk=1212958"
+    },
+    {
+      "pk": 1359073,
+      "url": "https://uptime.com/api/v1/checks/1359073/",
+      "name": "",
+      "cached_response_time": 15.440625,
+      "contact_groups": [
+        "Default",
+        "platform-services-product-owner"
+      ],
+      "created_at": "2022-09-14T21:20:43.140001Z",
+      "modified_at": "2022-09-26T20:44:13.894573Z",
+      "locations": [
+        "US East",
+        "US West",
+        "US Central"
+      ],
+      "tags": [
+        "shared-services"
+      ],
+      "check_type": "TRANSACTION",
+      "escalations": [],
+      "maintenance": {
+        "state": "ACTIVE",
+        "schedule": []
+      },
+      "monitoring_service_type": "TRANSACTION",
+      "is_paused": false,
+      "is_under_maintenance": false,
+      "state_is_up": true,
+      "state_changed_at": "2022-09-14T21:20:43.158020Z",
+      "msp_interval": 5,
+      "msp_address": "registry.developer.gov.bc.ca",
+      "msp_port": null,
+      "msp_protocol": "",
+      "msp_username": "",
+      "msp_proxy": "",
+      "msp_dns_server": "",
+      "msp_dns_record_type": "",
+      "msp_status_code": "",
+      "msp_send_string": "",
+      "msp_expect_string": "",
+      "msp_expect_string_type": "STRING",
+      "msp_encryption": "",
+      "msp_threshold": 60,
+      "msp_headers": "",
+      "msp_script": "[{\"step_def\":\"C_OPEN_URL\",\"values\":{\"url\":\"https://registry.developer.gov.bc.ca/\"}},{\"step_def\":\"C_WAIT_FOR_ELEMENT_TEXT\",\"values\":{\"element\":\"button\",\"text\":\"Login\",\"is_regex\":\"\"}},{\"step_def\":\"C_MOUSE_CLICK\",\"values\":{\"element\":\".css-1jhnpo2\",\"button\":\"left\",\"click_count\":\"1\",\"skip_navigation\":\"\"}},{\"step_def\":\"C_WAIT_FOR_ELEMENT\",\"values\":{\"element\":\"#password\"}},{\"step_def\":\"C_FILL_FIELD\",\"values\":{\"element\":\"#user\",\"text\":\"psta\",\"typing_delay\":\"\"}},{\"step_def\":\"C_FILL_FIELD\",\"values\":{\"element\":\"#password\",\"text\":\"p1aTf0rmS3rV!c3sT3Am\",\"typing_delay\":\"\"}},{\"step_def\":\"C_MOUSE_CLICK\",\"values\":{\"element\":\".login-form-action > .btn-primary\",\"button\":\"left\",\"click_count\":\"1\",\"skip_navigation\":\"\"}},{\"step_def\":\"C_WAIT_FOR_ELEMENT_TEXT\",\"values\":{\"element\":\".css-lgt0d3 .css-vurnku > .css-vurnku\",\"text\":\"Git documentation for Platform Services Registry\",\"is_regex\":\"\"}}]",
```

Comment:
you'll need to remove the credential for the IDIR account! Either revert commits and get rid of the history here, or you might have to update the test IDIR account credential.